spec/models/report_spec.rb

```diff
@@ -13,12 +13,12 @@
 
     @valid_post_report = {
       :item_id => @bob_post.id,
-      :item_type => 'post',
+      :item_type => 'Post',
       :text => 'offensive content'
     }
     @valid_comment_report = {
       :item_id => @bob_comment.id,
-      :item_type => 'comment',
+      :item_type => 'Comment',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.